### PR TITLE
Precalculus EQ5 Checkit tweeks

### PR DESCRIPTION
**Pull request by siwelwerd**
Addresses #300 

Fixes errors in answers and tweaks square root property task to include irrational roots.

### CODE REVIEW

**Comment by siwelwerd** [comment]
image

**Review by AbbyANoble — APPROVED** [review]
The example looks good, but I don't know enough about the code and what you changed to know if it works in general! We'll say yes for now. haha.

**Comment by siwelwerd** [comment]
>(Maybe this is better in the discussion than the pull request!)

Since this hasn't been merged yet, we can keep changing it here.  But after something is merged, then yes, discuss additional changes in a relevant issue (but crosslinking to related things like a recently merged PR is also helpful).

>I'm not sure that the first one is best by completing the square since it factors so easily.

Anything with rational roots is going to factor, so I made them irrational in the latest commit.

>The example looks good, but I don't know enough about the code and what you changed to know if it works in general! We'll say yes for now.

I also requested a review from @StevenClontz to eyeball the code for any glaring boneheaded mistakes.

File: source/precalculus/exercises/outcomes/EQ/EQ5/generator.sage [hunk 1]
```diff
@@ -19,25 +19,24 @@ class Generator(BaseGenerator):
     #Square root property
     a=choice([2..5])
     h=choice([-7..-1,1..7])
-    c=choice([2..5])
+    c=choice([sqrt(i) for i in [2..12]])
```
Comment (StevenClontz) [inline source/precalculus/exercises/outcomes/EQ/EQ5/generator.sage:22]:
```suggestion
    c=choice([sqrt(i) for i in [2,3,5,6,7,8,10,11,12]])
```

**Review by StevenClontz — APPROVED** [review]

**Comment by StevenClontz** [comment]
Merge or dismiss my suggestion, then this should be ready to squash and merge I think.

File: source/precalculus/exercises/outcomes/EQ/EQ5/generator.sage (hunk 1 above)
Comment (siwelwerd) [reply source/precalculus/exercises/outcomes/EQ/EQ5/generator.sage:22]: Maybe `c=choice([sqrt(i) for i in [2..12] if not i.is_square()])` is more readable?

Comment (siwelwerd) [reply source/precalculus/exercises/outcomes/EQ/EQ5/generator.sage:22]:
```suggestion
    c=choice([sqrt(i) for i in [2..12] if not i.is_square()])
```

**Pull request merged**